lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1671,7 +1676,8 @@ void LocalEnforcer::handle_set_session_rules(
     RuleSetBySubscriber rule_set_by_sub(rules_per_sub);
 
     for (const auto& session : session_it->second) {
-      auto& uc           = session_update[imsi][session->get_session_id()];
+      auto session_id    = session->get_session_id();
+      auto& uc           = session_update[imsi][session_id];
```

Comment:
fix this  on a second commit on this PR too

[second commit](https://github.com/magma/magma/pull/10211/commits/48ac91cc609a9cf8553ad0e0dddbe046c1f95193)